Zend/zend_API.h

```diff
@@ -1256,7 +1256,8 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_argument_value_error(uint32_t arg_num
 #define ZEND_PARSE_PARAMETERS_START_EX(flags, min_num_args, max_num_args) do { \
 		const int _flags = (flags); \
 		uint32_t _min_num_args = (min_num_args); \
-		int _max_num_args = (max_num_args);  /* TODO uint32_t */ \
+		int __max_num_args = (max_num_args); \
+		uint32_t _max_num_args = __max_num_args >= 0 ? __max_num_args : UINT32_MAX; \
```

Comment:
Just `uint32_t _max_num_args = (uint32_t) max_num_args` should be sufficient.

Reply:
Done.
But in my opinion, we should avoid some hacks (although it's tiny and we know it won't be a problem)
For example, if someone uses a pointer as max_num_args accidentally, there will be no warnings to remind him

Reply:
Casting a pointer to uint32_t will give a warning :)

Reply:
Sorry, I forgot that this needs to be converted to uintptr_t first, it seems that my worries are unnecessary🤪
(But my clang does not give any warning, gcc is fine)